pkg/testutils/lint/gcassert_paths.txt

```diff
@@ -21,6 +21,7 @@ sql/colexec/colexecsel
 sql/colexec/colexecspan
 sql/colexec/colexecwindow
 sql/colfetcher
+sql/execinfrapb
```

Comment:
Is this change enough to ensure that `gcassert` is run on the `execinfrapb` package?